### PR TITLE
add .env.dev and .env.dev.local as possible env file

### PR DESCRIPTION
**Pull request by Fabex**
This PR allow 2 more files as possible env file

I add .env.dev and .env.dev.local

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 - Added several Relay built-in directives.
 - Supported suppression of inspections with comment for the definition or the whole file.
 - Changing the file highlight level in the editor prevents any inspections from being displayed.
-- Load variables from .env files. Supported file names: `.env.local`,`.env.development.local`,`.env.development`,`.env`.
+- Load variables from .env files. Supported file names: `.env.local`,`.env.development.local`,`.env.development`,`.env.dev.local`,`.env.dev`,`.env`.
```
Comment (vepanimas) [inline CHANGELOG.md:17]: Please, revert it. It's a changelog for the already released version.

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ GraphQL language support for WebStorm, IntelliJ IDEA and other IDEs based on the
 - Syntax highlighting, code-formatting, folding, commenter, and brace-matching
 - 'Find Usages' and 'Go to Declaration' for schema types, fields, and fragments
 - 'Structure view' to navigate GraphQL files
-- Load variables from .env files. Supported file names: `.env.local`,`.env.development.local`,`.env.development`,`.env`
+- Load variables from .env files. Supported file names: `.env.local`,`.env.development.local`,`.env.development`,`.env.development.local`,`.env.development`,`.env`
```
Comment (vepanimas) [inline README.md:18]: `.env.development.local,.env.development` are duplicated

**Review by vepanimas — CHANGES_REQUESTED** [review]
LGTM, let's merge after fixing the minor issues.

File: CHANGELOG.md (hunk 1 above)
Comment (vepanimas) [reply CHANGELOG.md:17]: You could add a new entry for the next version 3.1.0 like 
```
## [3.0.0] - 2021-07-16

### Added
- <description of your change>
```

Comment (Fabex) [reply CHANGELOG.md]: done

File: README.md (hunk 2 above)
Comment (Fabex) [reply README.md]: done

**Review by vepanimas — APPROVED** [review]

**Comment by vepanimas** [comment]
@Fabex merged 🎉 , thank you for the contribution!

**Pull request merged**